workflows/snakemake/README.md

```diff
@@ -0,0 +1,66 @@
+# Snakemake
+
+```
+# Install micromamba on snellius
+"${SHELL}" <(curl -L micro.mamba.pm/install.sh)
+# During install, set conda-forge as default channel
```

Comment:
I didn't get a prompt for this, and went through the default options. Is that the correct way to do this?

Reply:
I guess it doesn't make much difference. You should always be able to specify conda-forge later.